device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,285 @@
+{
+    "chassis": {
+        "name": "MSN2700",
+        "components": [],
+        "fans": [
+            {
+                "name": "fan1"
+            },
+            {
+                "name": "fan2"
+            },
+            {
+                "name": "fan3"
+            },
+            {
+                "name": "fan4"
+            },
+            {
+                "name": "fan5"
+            },
+            {
+                "name": "fan6"
+            },
+            {
+                "name": "fan7"
+            },
+            {
+                "name": "fan8"
+            }
+        ],
+        "fan_drawers": [
+            {
+                "name": "drawer1",
+                "num_fans": 2
```

Comment:
Rather than `num_fans`, I believe we should change this to:

```
"fans": [
    {
        "name": "fan_1"
    },
    {
        "name": "fan_2"
    }
]
```

@keboliu: Please confirm.

Reply:
we do have "num_fans" here, and the name should be "fanx" w/o '_'

Reply:
Thanks, @keboliu. FYI, the API currently returns fan names _with_ the underscore. So it appears this is another thing which will need to be updated in your API, along with the chassis name.

Reply:
fixed